homeassistant/components/mqtt/mixins.py

```diff
@@ -244,9 +244,11 @@ def validator(config: ConfigType) -> ConfigType:
             return config
 
         _LOGGER.error(
-            "Manually configured MQTT %s(s) found under platform key '%s', "
-            "please move to the mqtt integration key, see "
-            "https://www.home-assistant.io/integrations/%s.mqtt/#new_format",
+            (
```

Comment:
Same here, strings become larger.

Reply:
So, it uses some additional parentheses when parameters are involved.

This separates the actual string from the parameters.

If there are no additional parameters, the parentheses will be omitted.